pytorch/training/docker/2.0/py3/cu118/Dockerfile.gpu

```diff
@@ -285,6 +286,11 @@ RUN /opt/conda/bin/mamba install -y -c conda-forge \
   cmake \
   lit
 
+# Patches
+RUN pip install "pillow>=9.5"
```

Comment:
using pip as mamba doesn't resolve the dependencies and throws conflicts error

Reply:
can you add more context, why cant we bump up the existing pillow version in the above conda installation? 
We had issues before when installing packages from both pip and conda. lets stick the installation to conda channel

Reply:
If continuing with the approach of performing a pip install for Pillow, please also insert this line after the SM PyTorch installation step as well.